lra/lrademo/trip-manager-springboot/pom.xml

```diff
@@ -58,13 +58,6 @@
             <artifactId>microtx-lra-spring-boot-starter</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
-        
```

Comment:
Even though library has this dependency, Application still needs to add this dependency for sample app representation. Let's not remove it.

Reply:
Ok Bharath